### ISSUE TITLE
Move `dido` back to `io2` EBS Storage Type

### ISSUE THREAD

**Issue by ischasny** [ipni/storetheindex#1459]
**TLDR; my proposal is to change `dido`'s EBS Storage Type back to `io2-iops5` as `gp3` has  much worse random reads profile for mixed read / write workloads - 2x on average but much worse in extremes (what we are observing in prod with p99s). More thoughts below.** 

Recently we've observed high read latencies on `dido` that in its turn resulted to a number of High Upstream Latency alerts in `prod`.

![image](https://user-images.githubusercontent.com/31857042/227173464-b3961e79-afad-4463-86ef-437d4eec795f.png)

`dido`'s latencies were 10-100x comparing to other nodes. To get to the bottom of it I've checked a few things.

1. Rolled out pebble metrics to `prod`. Pebble can perform poorly on reads and writes when its LSM becomes [inverted](https://www.cockroachlabs.com/docs/stable/architecture/storage-layer.html#inverted-lsms). The symptom of inverted LSM are high `Read Amplification` (higher than 50 for longer than 1 hour) and a large number of open `L0 File Descriptors` (number in high thousands). None of these were the case for `dido`. More on monitoring [here](https://www.cockroachlabs.com/docs/stable/common-issues-to-monitor.html#lsm-health).

![image](https://user-images.githubusercontent.com/31857042/227174895-24a3a0ed-98c9-4ae0-b501-49d3b0fec010.png)
![image](https://user-images.githubusercontent.com/31857042/227174974-4a67b175-f699-43b6-88ea-ece2e59ee2df.png)

2. Increased Pebble Block Cache size from `1Gi` to `2Gi` to see whether it improves the read performance. Pebble used up all available cache space without any noticeable effect on the latencies.  

![image](https://user-images.githubusercontent.com/31857042/227175425-54a7c27b-aa15-4d64-b540-e9d38febab6f.png)

3. Can `dido` be starving on RAM or CPU? I've compared RAM and CPU profiles with another instance called `kepa` that doesn't have same issues with read latencies. Both instances have similar profiles.

![image](https://user-images.githubusercontent.com/31857042/227176703-77e28460-d600-40b1-93e1-0bbf860b3597.png)
![image](https://user-images.githubusercontent.com/31857042/227176790-b4b4ea89-299a-40d2-9856-8eba42d36e09.png)

4. `dido` and `kepa` use different EBS storage types. `dido` uses `gp3` while `kepa` is on `io2`. `gp3` (top) shows 2x average read latency comparing to `io2`(bottom).

![image](https://user-images.githubusercontent.com/31857042/227178890-e2515b4e-a1a9-4c8f-bd51-d8c28f116edd.png)
![image](https://user-images.githubusercontent.com/31857042/227178970-569efae1-d2bb-4069-b450-ff3ab6f5b56f.png)

However more importantly, `io2` has much flatter latency distribution for [mixed read/write workloads](https://www.percona.com/blog/performance-of-various-ebs-storage-types-in-aws/). Even though average read latency might have 2x difference the extremes are much worse, which I believe what we are observing in `prod`.

----

`dido` was [moved](https://github.com/ipni/storetheindex/commit/c24d7182374fe9dad5b639caf5e194b7d6188e71) to `gp3` on 14th February 2023 which had an immediate effect on its read latencies. Recent increase in Rhea read traffic has made `gp3` read latency extremes worse. 

![image](https://user-images.githubusercontent.com/31857042/227185348-ab7d28ad-73fb-4705-b04c-e4bbbd7e1383.png)

**Comment by ischasny** [comment]
CC @gammazero @masih @davidd8

**Comment by masih** [comment]
Have we tried bumping throughput on dido's gp3 volume?

**Comment by ischasny** [comment]
> Have we tried bumping throughput on dido's gp3 volume?

Happy to try but it doesn't look like throughput is the problem. The problem is very inconsistent read latency on `gp3`.

See bandwidths for `dido` and `kepa` that are roughly the same:

![image](https://user-images.githubusercontent.com/31857042/227284105-434a4545-cb39-4074-88bf-57eea15f91f0.png)

![image](https://user-images.githubusercontent.com/31857042/227284972-fcde884b-04a7-424b-bcc2-fe90469e9ea2.png)

**Comment by masih** [comment]
We should also try AWS support on this. It would be great to understand: 
1. why it is happening now, and
2. why it happens for the same IO operations when time spent idle for `dido` gp3 volume is about 40%

**Comment by masih** [comment]
I think we can close this now that `dido` is performing with `gp3` volume of 5K IOPS and 300 MiB throughput.
